.eslintrc

```diff
@@ -25,6 +25,7 @@
   "rules": {
     "linebreak-style": ["error", "unix"],
     "require-atomic-updates": "off",
+    "cypress/no-unnecessary-waiting": "off",
```

Comment:
Do we still need this default override now we're using `wait` on the requests rather than the time? I think we should stick to the defaults if we don't need to change this after all.